tests/snapshots/tests.contrib.ray.test_ray.error_in_task.json

```diff
@@ -0,0 +1,574 @@
+[[
+  {
+    "name": "ray.job",
+    "service": "raysubmit_p4qyzGF2UjAseg8C",
```

Comment:
Looking at a few of the service names in this snapshot, these service names seem to include a very high cardinality string towards the end. Can the default be `raysubmit` somehow?